mlflow/java/client/src/main/java/org/mlflow/tracking/MlflowClient.java

```diff
@@ -658,9 +658,10 @@ private ArtifactRepository getArtifactRepository(String runId) {
    * The current available stages are: [None, Staging, Production, Archived].
    *
    *    <pre>
-   *        List<ModelVersionDetailed> detailsList = getLatestVersions("model");
```

Comment:
The underlying cause of the doc-build failures that [this change](https://github.com/mlflow/mlflow/pull/2308/files#diff-037c7ca4cfb39d9c9d82de298b20cab0L83) aimed to mitigate was malformed Javadocs.

You can reproduce the errors by checking out master, adding back `org.mlflow.tracking` to the set of included Javadoc packages in mlflow/java/client/pom.xml, and running `mvn clean javadoc:javadoc` from the `mlflow/java` directory:

```
[ERROR] /Users/sid.murching/code/mlflow/mlflow/java/client/src/main/java/org/mlflow/tracking/MlflowClient.java:661: error: unknown tag: ModelVersionDetailed
[ERROR]    *        List<ModelVersionDetailed> detailsList = getLatestVersions("model");
[ERROR]                 ^
[ERROR] /Users/sid.murching/code/mlflow/mlflow/java/client/src/main/java/org/mlflow/tracking/MlflowClient.java:688: error: unknown tag: ModelVersionDetailed
[ERROR]    *        List<ModelVersionDetailed> detailsList  = getLatestVersions("model",
[ERROR]                 ^
[ERROR] /Users/sid.murching/code/mlflow/mlflow/java/client/src/main/java/org/mlflow/tracking/MlflowClient.java:689: error: malformed HTML
[ERROR]    *                                                              Lists.newArrayList<>("Staging"));
[ERROR]                                                                                     ^
[ERROR] /Users/sid.murching/code/mlflow/mlflow/java/client/src/main/java/org/mlflow/tracking/MlflowClient.java:689: error: bad use of '>'
[ERROR]    *                                                              Lists.newArrayList<>("Staging"));
[ERROR]                                                                                      ^
[ERROR] /Users/sid.murching/code/mlflow/mlflow/java/client/src/main/java/org/mlflow/tracking/MlflowClient.java:747: error: no tag name after @
[ERROR]    * @return A local file or directory ({@ java.io.File}) containing model artifacts
[ERROR]                                         ^
[ERROR] /Users/sid.murching/code/mlflow/mlflow/java/client/src/main/java/org/mlflow/tracking/MlflowClient.java:774: error: no tag name after @
[ERROR]    * @return A local file or directory ({@ java.io.File}) containing model artifacts
```

The changes below fix the abovementioned issues